man/systemd.exec.xml

```diff
@@ -2134,6 +2134,39 @@ BindReadOnlyPaths=/var/lib/systemd</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MemoryTHP=</varname></term>
+
+        <listitem><para>Transparent Hugepages (THPs) is a Linux kernel feature that manages memory
+        using larger pages (2MB on x86, compared to the default 4KB). The main goal is to improve memory management
+        efficiency and system performance, especially for memory-intensive applications.
+        However, it can cause drawbacks in some scenarios, such as memory regression and latency spikes.
+        THP policy is governed for the entire system via <filename>/sys/kernel/mm/transparent_hugepage/enabled</filename>.
+        However, it can be overridden for individual workloads via
+        <citerefentry><refentrytitle>prctl</refentrytitle><manvolnum>2</manvolnum></citerefentry>.
+        MemoryTHP= may be used to disable THPs at process invocation time to stop providing THPs for workloads where
+        the drawbacks outweigh the advantages.
+        When MemoryTHP is set to <literal>default</literal> or not set at all, systemd inherits THP settings from the process
+        that starts it and no <citerefentry><refentrytitle>prctl</refentrytitle><manvolnum>2</manvolnum></citerefentry>
+        PR_SET_THP_DISABLE call is made.
+        When set to <literal>disable</literal>, MemoryTHP disables THPs completely for the process, irrespecitive
```

Comment:
when you reference the setting please always suffix with "=" to make clear you are talking about the setting. we do this for all our settings in docs + log messages, hence please do so here too.

also enclose in <varname> here in docbook